<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>roomDialog</class>
 <widget class="QDialog" name="roomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="wdg_title" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 222, 222);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pb_image">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>^_^</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_title">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;房间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="buddy">
       <cstring></cstring>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>213</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pb_min">
      <property name="minimumSize">
       <size>
        <width>31</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>31</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_close">
      <property name="minimumSize">
       <size>
        <width>31</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>31</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>X</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pb_day">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>93</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>第几天</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_num">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>房间号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_roomNum">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>000000</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_type">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>房间类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_roomType">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>xx人xx板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_set">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>40</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="roomPlayerform" name="wdg_left" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>90</width>
     <height>500</height>
    </rect>
   </property>
  </widget>
  <widget class="roomPlayerform" name="wdg_right" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>90</width>
     <height>500</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_quitroom">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出房间</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>roomPlayerform</class>
   <extends>QWidget</extends>
   <header>roomplayerform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
